references.bib

```diff
@@ -80,3 +72,18 @@ @book{james2013introduction
   publisher={Springer}
 }
 
+@Manual{palmerpenguins,    
+    title = {palmerpenguins: Palmer Archipelago (Antarctica) penguin data},
```

Comment:
This gets typeset as `Palmerpenguins` in the textbook, it should be `palmerpenguins`. You can achieve this by adding curly braces around anything you want to preserve the formatting of, e.g.

```
    title = {{palmerpenguins}: Palmer Archipelago (Antarctica) penguin data},
```